Lib/test/test_sys.py

```diff
@@ -850,7 +850,9 @@ def test_sys_getwindowsversion_no_instantiation(self):
 
     @test.support.cpython_only
     def test_clear_type_cache(self):
-        sys._clear_type_cache()
+        with self.assertWarnsRegex(DeprecationWarning,
+            r"sys._clear_type_cache\(\) is deprecated.*"):
```

Comment:
Function arguments should be aligned if the first is on the same line as the function:
```suggestion
        with self.assertWarnsRegex(DeprecationWarning,
                                   r"sys._clear_type_cache\(\) is deprecated.*"):
```